### PR TITLE
Added files via upload

### PR DESCRIPTION
**Pull request by ruihanzou**
初稿

### CODE REVIEW

File: 【太阁X】第十期：聊聊技术博客的那些事儿.md [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+
+【董飞】
+
+热爱技术，相信使命，追求极致。先后创业公司酷迅，百度基础架构组，Amazon云计算部门，Linkedin担任高级工程师，负责过垂直搜索，百度云计算平台研发和广告系统的架构。目前就职于在线教育创业公司Coursera。
+
+- 微信公众号：董老师在硅谷 donglaoshi-123
+- 知乎专栏：董老师在硅谷 http://zhuanlan.zhihu.com/donglaoshi
+- 百度百家专栏：http://dongfei.baijia.baidu.com/
+- Facebook Page：董老师说码工的那点事儿 
+- https://www.facebook.com/donglaoshi123
```
Comment (Ishiihara) [inline 【太阁X】第十期：聊聊技术博客的那些事儿.md]: This line should be combined with the line above.

File: 【太阁X】第十期：聊聊技术博客的那些事儿.md [hunk 2]
```diff
@@ -0,0 +1,191 @@
...
+【魏楚阳】 
+
+电信专业，华科研一。目前的方向是web开发和网络爬虫。很喜欢BitTiger大家庭的氛围，技术视野变宽了，同时认识了很多优秀的小伙伴。虽然目前还是小白，但对技术有追求，对学习有热情，希望可以快速成长为和大家一样优秀的ITer！
+
+- LinkedIn： https://cn.linkedin.com/in/weichuyang/en
+- 个人网站：http://brianway.github.io
+- CSDN：http://blog.csdn.net/h3243212
+- oschina：http://my.oschina.net/brianway
+
+【王思唯】优秀脱口秀主持人
```
Comment (Ishiihara) [inline 【太阁X】第十期：聊聊技术博客的那些事儿.md]: 优秀脱口秀主持人 should be put below to match the introduction of other speakers.

File: 【太阁X】第十期：聊聊技术博客的那些事儿.md [hunk 3]
```diff
@@ -0,0 +1,191 @@
...
+
+本科金融，目前在Emory读计算机硕士。计算机知识全靠自学。自从加入了BitTiger大家庭，简历好看了，面试轻松拿到了，技能学会了，更重要的是认识了一大帮子超级优秀的人。只有人比人才能督（qi）促( si)人。所以欢迎各位加入BitTiger，让我们一起愉(zuo)快(si)的学习新知识吧。个人暂时能拿出手的是机器学习技术，目标是成为一名全栈工程师。
+
+- LinkedIn: https://www.linkedin.com/in/siwei-wang-5289a6a4/en
+
+
+## 主要内容
+
+### 董老师的技术博客之旅(by董飞)
```
Comment (Ishiihara) [inline 【太阁X】第十期：聊聊技术博客的那些事儿.md]: remove by.

File: 【太阁X】第十期：聊聊技术博客的那些事儿.md [hunk 4]
```diff
@@ -0,0 +1,191 @@
...
+
+这部分主要是按照不同的类别，推介一些优秀的博文。包括列表类，创业类，翻译类，等等。
+
+**列表类**
+
+董老师微信公众号分类列表(http://mp.weixin.qq.com/mp/homepage?__biz=MzA3NTM4NDE2Mw==&hid=1&sn=dfec54dc8ee4a5d76e455d90a827aec4#wechat_redirect)
```
Comment (Ishiihara) [inline 【太阁X】第十期：聊聊技术博客的那些事儿.md]: All links should be in the reference section. Only reference the label here.

File: 【太阁X】第十期：聊聊技术博客的那些事儿.md [hunk 5]
```diff
@@ -0,0 +1,191 @@
...
+
+以[《100家硅谷IT公司技术博客》](http://mp.weixin.qq.com/s?__biz=MzA3NTM4NDE2Mw==&mid=401398776&idx=1&sn=76a3c474ab4d3b9d3b732b9c5dffc619&scene=21#wechat_redirect)为例，列举了其中一些好的博客
```
Comment (Ishiihara) [inline 【太阁X】第十期：聊聊技术博客的那些事儿.md]: 董老师整理了《100家硅谷IT公司技术博客。 其中涵盖了很多大公司。。。

File: 【太阁X】第十期：聊聊技术博客的那些事儿.md [hunk 6]
```diff
@@ -0,0 +1,191 @@
...
+- 知名大厂，如：LinkedIn,Netflix，AWS，Baidu Research，Yahoo，Twitter.
+    - [《首席工程师揭秘：LinkedIn大数据后台是如何运作的》](http://mp.weixin.qq.com/s?__biz=MzA3NTM4NDE2Mw==&mid=401350558&idx=3&sn=f84a1335a3dc31753f3ea496fc2a87b6&scene=21#wechat_redirect)([英文原文](https://engineering.linkedin.com/distributed-systems/log-what-every-software-engineer-should-know-about-real-time-datas-unifying))。图文并茂，对数据库、分布式、系统设计都有涉及，非常值得大家阅读。
+    - AWS,在云计算方面比较有名，参考[AWS Blog](https://aws.amazon.com/cn/blogs/aws/)
+    - Twitter,open source比较有名，参考[Open Source at Twitter](https://engineering.twitter.com/opensource)
+
+- 热门创业公司,如：Uber,AirBnB,Pinterest,Slack,Databricks,Confluent
+  - AirBnB,其[Tech Talks](http://nerds.airbnb.com/tech-talks/)比较有名,有自己的视频，可以听一听博客。
+  - Pinterest是做图片兴趣社交的，参考[Open Sources - Pinterest Engineering](https://engineering.pinterest.com/open-source)
+  - Databricks,做spark内存计算公司
+  - Confluent,kafaka的几个元老出来做的公司。参考[Confluent Blog](http://www.confluent.io/blog)
```
Comment (Ishiihara) [inline 【太阁X】第十期：聊聊技术博客的那些事儿.md]: kafaka -> Apache Kafka
元老 -> 主要开发者
remove 参考

File: 【太阁X】第十期：聊聊技术博客的那些事儿.md [hunk 7]
```diff
@@ -0,0 +1,191 @@
...
+[2015年斯坦福创业课程](https://github.com/dongfeiwww/blitzscaling)
+
+**翻译类**
+
+[2015年科技阅读列表600篇](http://zhuanlan.zhihu.com/p/20386800)
+
+**技术类**
+
+High Scalability，http://highscalability.com/,介绍各个公司的架构
+
```
Comment (Ishiihara) [inline 【太阁X】第十期：聊聊技术博客的那些事儿.md]: Do we need the empty lines here?

File: 【太阁X】第十期：聊聊技术博客的那些事儿.md [hunk 8]
```diff
@@ -0,0 +1,191 @@
...
+- 3.总结提升
+
+把笔记升华为博客
+
+- 要有**图**，一图胜千言。
+- 以问题为中心(what,why,how)
+- 层次段落要清晰(标题，子标题，章节)
+- 突出中心(记录解决问题、如何解决而非记录解决问题花费的功夫),节约读者时间
+
+示例：什么是Counting Bloom Filter？http://blog.csdn.net/jiaomeng/article/details/1498283
```
Comment (Ishiihara) [inline 【太阁X】第十期：聊聊技术博客的那些事儿.md]: Need to talk about why this is good.

File: 【太阁X】第十期：聊聊技术博客的那些事儿.md [hunk 9]
```diff
@@ -0,0 +1,191 @@
...
+现有平台
+
+- CSDN：号称“全球最大中文IT社区”
+- oschina：开源中国。是目前中国最大的开源技术社区。
+- SegmentFault：更像一个问答社区。
+- github:可以直接新建一个repo，然后上传自己的博客。
+
+搭建个站
+
+github pages，常用博客引擎有：
```
Comment (Ishiihara) [inline 【太阁X】第十期：聊聊技术博客的那些事儿.md]: It would be helpful to provide a link to a tutorial on how to build a personal blog platform.

**Comment by Ishiihara** [comment]
@ruihanzou Left some comments. I think the major issue here is that this is more like a meeting notes instead of a blog post. We probably need to add some transition sentences and some summaries in each section to make it more readable.

**Comment by Ishiihara** [comment]
@ruihanzou Also, the name of the PR should be the blog name.

File: 【太阁X】第十期：聊聊技术博客的那些事儿.md [hunk 10]
```diff
@@ -0,0 +1,191 @@
...
+
+- 知名大厂，如：LinkedIn,Netflix，AWS，Baidu Research，Yahoo，Twitter.
+    - [《首席工程师揭秘：LinkedIn大数据后台是如何运作的》](http://mp.weixin.qq.com/s?__biz=MzA3NTM4NDE2Mw==&mid=401350558&idx=3&sn=f84a1335a3dc31753f3ea496fc2a87b6&scene=21#wechat_redirect)([英文原文](https://engineering.linkedin.com/distributed-systems/log-what-every-software-engineer-should-know-about-real-time-datas-unifying))。图文并茂，对数据库、分布式、系统设计都有涉及，非常值得大家阅读。
+    - AWS,在云计算方面比较有名，参考[AWS Blog](https://aws.amazon.com/cn/blogs/aws/)
+    - Twitter,open source比较有名，参考[Open Source at Twitter](https://engineering.twitter.com/opensource)
+
+- 热门创业公司,如：Uber,AirBnB,Pinterest,Slack,Databricks,Confluent
+  - AirBnB,其[Tech Talks](http://nerds.airbnb.com/tech-talks/)比较有名,有自己的视频，可以听一听博客。
+  - Pinterest是做图片兴趣社交的，参考[Open Sources - Pinterest Engineering](https://engineering.pinterest.com/open-source)
+  - Databricks,做spark内存计算公司
```
Comment (Ishiihara) [inline 【太阁X】第十期：聊聊技术博客的那些事儿.md]: Databricks blog link: https://databricks.com/blog. 
spark -> Spark 
Spark的主要作者成立的公司。主要产品是Spark Cloud。

Comment (Ishiihara) [inline 【太阁X】第十期：聊聊技术博客的那些事儿.md]: Databricks Cloud.

**Comment by Ishiihara** [comment]
@ruihanzou Also we need a directory for each blog.

File: 【太阁X】第十期：聊聊技术博客的那些事儿.md [hunk 11]
```diff
@@ -0,0 +1,191 @@
...
+---
+[TOC]
+
+---
+
+## 嘉宾介绍
+
+【董飞】
+
+热爱技术，相信使命，追求极致。先后创业公司酷迅，百度基础架构组，Amazon云计算部门，Linkedin担任高级工程师，负责过垂直搜索，百度云计算平台研发和广告系统的架构。目前就职于在线教育创业公司Coursera。
```
Comment (hackjustu) [inline 【太阁X】第十期：聊聊技术博客的那些事儿.md]: `先后创业公司酷迅...` -> `先后在创业公司酷迅...`

**Pull request merged**